pil-analyzer/src/pil_analyzer.rs

```diff
@@ -474,6 +478,9 @@ impl PILAnalyzer {
                     match item {
                         PILItem::Definition(symbol, value) => {
                             let name = symbol.absolute_name.clone();
+                            if matches!(value, Some(FunctionValueDefinition::TraitDeclaration(_))) {
+                                self.trait_impls.entry(name.clone()).or_default();
```

Comment:
Is there any reason to add the name here? (the vector may be empty if it has no implementation)